package.json

```diff
@@ -28,7 +28,7 @@
     "tsc:esm": "tsc --module es2015 --outDir esm",
     "tsc:commonjs": "tsc --module commonjs --outDir lib",
     "semantic-release": "semantic-release",
-    "test": "jest"
+    "test": ""
```

Comment:
:i-can-fix-it:

Reply:
![image](https://user-images.githubusercontent.com/64043462/221256973-c6c402cf-8351-4bbf-b6c7-c4175edaa6e2.png)